src/main/g8/src/main/scala/$package$/CborSerializable.scala

```diff
@@ -0,0 +1,7 @@
+package $package$
+
+/**
+ * Marker trait for serialization with Jackson CBOR.
+ * Enabled in serialization.conf `akka.actor.serialization-bindings` (via application.conf).
```

Comment:
```suggestion
 * Enabled in serialization.conf `akka.actor.serialization-bindings` (via serialization.conf).
```

Reply:
via refers to the include from application.conf